src/PrestaShopBundle/Resources/config/services/adapter/data_configuration.yml

```diff
@@ -236,3 +236,23 @@ services:
         arguments:
             - '@prestashop.adapter.legacy.configuration'
             - '@=service("prestashop.adapter.data_provider.default_route").getRules()'
+
+    prestashop.adapter.webservice.webservice_account_eraser:
+        class: 'PrestaShop\PrestaShop\Adapter\Webservice\WebserviceAccountEraser'
```

Comment:
What are these classes ?

Reply:
Hmm I have no clue actually, probably came with the rebase? @tomas862 since you migrated the webservices page, wdyt?

Reply:
its an adapter that @tomas862 added instead of command, it should be removed at some point when we implement CQRS for it.

Reply:
these classes don't even exist, only the service declarations are there. I'll remove them.